### ISSUE TITLE
Read in batch

### ISSUE THREAD

**Issue by xpepermint** [neumino/rethinkdbdash#187]
I need to scroll through ~10 mio documents thus I need to read data in batches by ~1000 docs per batch. I don't see an option to set the `maxBatchRows` and I don't even know how to test this if it rely works. I guess I need to call my query with `run({cursor: true})` to get a `cursor` and then use `cursor.each` to loop through the results. I imagine that if I don't use `cursor` all results will be retrieved as a single batch.

Example:
```js
let cursor = yield r.table(`logs`).filter({tag: 'error'}).run({cursor: true});
cursor.each((err, row) => {
  if (err) throw err;
  console.log(row);
});
```

Will this code work? How can I set the `maxBatchRows` option?

**Comment by neumino** [comment]
It was an internal option only before. I didn't know they made it as a plain option for 2.2.

It's available in 2.2.2.

**Comment by xpepermint** [comment]
You are awesome!
